apps/sms/test/unit/thread_ui_test.js

```diff
@@ -5225,7 +5239,9 @@ suite('thread_ui.js >', function() {
       Navigation.isCurrentPanel.withArgs('composer').returns(true);
 
       // Please, auto-answering stub, don't interfere with my test
-      this.sinon.stub(Contacts, 'findExact');
+      this.sinon.stub(Contacts, 'findExact').returns({
+        then: () => {}
```

Comment:
tell me if the new code looks better for you, otherwise I can do your promise-that-never-resolves.